docs/examples/tensorflow/tensorflow-resnet50-various-readers.ipynb

```diff
@@ -84,7 +88,20 @@
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python2.7/dist-packages/nvidia/dali/backend.py:15: FutureWarning: pybind11-bound class 'nvidia.dali.backend_impl.TensorCPU' is using an old-style placement-new '__init__' which has been deprecated. See the upgrade guide in pybind11's docs. This message is only visible when compiled in debug mode.\n",
+      "  from nvidia.dali.backend_impl import *\n",
+      "/usr/local/lib/python2.7/dist-packages/nvidia/dali/backend.py:15: FutureWarning: pybind11-bound class 'nvidia.dali.backend_impl.TensorListCPU' is using an old-style placement-new '__init__' which has been deprecated. See the upgrade guide in pybind11's docs. This message is only visible when compiled in debug mode.\n",
+      "  from nvidia.dali.backend_impl import *\n",
+      "/usr/local/lib/python2.7/dist-packages/nvidia/dali/backend.py:15: FutureWarning: pybind11-bound class 'nvidia.dali.backend_impl.TensorListGPU' is using an old-style placement-new '__init__' which has been deprecated. See the upgrade guide in pybind11's docs. This message is only visible when compiled in debug mode.\n",
+      "  from nvidia.dali.backend_impl import *\n"
+     ]
+    }
+   ],
```

Comment:
This is output from debug build of DALI, please remove it

Reply:
Done